### ISSUE TITLE
Phase I

### ISSUE THREAD

**Issue by ignacioFernandez1** [kbroughton/spell_chornk#5]
The Phase I of the program should change random letters in a given number of words of a text with the --num-changes option (see issue #1 ). It should have a --text "string or @path/to/file" where the @ indicates that the input is a file. The ouput of the program should be a json like the one in #1
Once this is done we could add a dict or bidict with common mis-spellings to replace those words with the mis-spelling. A stretch goal would be to use abbreviations instead of mis-spellings.
